docs/reference/configurations.mdx

```diff
@@ -262,7 +264,7 @@ export default {
       supports: "*",
       config: {
         host: "redis",
-        // host: "127.0.0.1"
+        // full configuration at https://developers.front-commerce.com/docs/advanced/graphql/dataloaders-and-cache-invalidation#redis
```

Comment:
I hesitated to paste here the full configuration too :thinking: any opinion ?

Reply:
Maybe put it as a link inside a  `:::info` block below the code snipet ? I think it might make it more visible than inside a comment (if the redis part is the only important part here).

Reply:
I would say it becomes a bit too visible in an info (this current setting will nearly never work as is for the developer, leading to follow the link in the code block to configure it properly)